app/src/main/java/infrmr/newsapp/github/com/ifrmr/MainActivity.java

```diff
@@ -185,7 +192,7 @@ protected void onPostExecute(String result) {
     }
 
 
-    // Uploads XML from stackoverflow.com, parses it, and combines it with
+    // Uploads XML from TheVerge.com, parses it, and combines it with
     // HTML markup. Returns HTML string.
```

Comment:
You can add in the comments in javadoc syntax. Type out "/**" and hit enter in studio, it'll generate a javadoc comment block for you.